src/api/controllers/articleController.js

```diff
@@ -270,4 +301,68 @@ export default class ArticleController {
       errorSender(BAD_REQUEST, res, 'text', errorMessage.textMatch);
     }
   }
+
+  /**
+   *
+   * search for artcile according to user's request
```

Comment:
@alainburindi search for `an article`

Reply:
@alainburindi you haven't implemented this feedback. Check your spelling of `article`